website/content/community/_index.md

```diff
@@ -145,6 +145,18 @@ the [github repository](https://github.com/google/metallb), and add
 your fork as a remote in `$GOPATH/src/go.universe.tf/metallb`, with
 `git remote add fork git@github.com:<your-github-user>/metallb.git`.
 
+## Testing a Pull Request
+
+MetalLB uses CI to publish images of the latest version of code from each open
+pull request.  If you would like to test out a prebuilt image of a PR, make
+sure the `deploy-speaker` or `deploy-controller` jobs have succeeded on the PR,
+and then you may pull `metallbdev/speaker:pr-<ID>` or
+`metallbdev/controller:pr-<ID>`.
+
+If a new revision of a pull request is posted, the image will be rebuilt.  If
```

Comment:
```suggestion
If a new revision of a pull request is posted, the image will be rebuilt. If
```

Looks like your editor consistently inserts double spaces after a period :thinking: Could you please fix all occurrences?

Reply:
My thumb does it.  ;-)

I'll change them.